### DISCUSSION TITLE
Old Notes tab interface?

### DISCUSSION THREAD

**Discussion by ghost** [AprilSylph/XKit-Rewritten#469] [Suggestion Box]
So the new update has made it incredibly obnoxious to see all the notes on posts. What I used to be able to look over with one click I now have to click through four different tabs to check and scroll back up to press a button to close again, defaulting to the comments tab regardless of which type of notes the post actually has. I don't know what would actually be feasible to do, but an extension to either restore something like the functionality of the old notes interface or at least add a tab to the new one that displays all types of notes at once would be a massive quality of life improvement.

**Comment by AprilSylph** [comment]
Can't be done.
